### PR TITLE
Add Convert to Celsius

### LINKED ISSUE #9: Convert to Celsius
# 5. Convert to Celsius

> assigned to `Aleksandra`, reviewers `Oguz` and `Mamé`

### Repository

- [x] This user story was developed locally on a brach called `convert-to-celsius`.
- [x] The new branch was pushed to the shared repository
- [x] A pull request was made to `master` & reviewed by the team
- [x] Changes were merged to `master`

### HTML

- [x] article2 element that includes:
   - [x] header with the name of the city
   - [x] instruction paragraph
   - [x] label and input for the argument of the function
   - [x] span with the result of the function

### CSS

- [x] Style of the article2

## JS
- [x] function convertToCelsius with argument linked to HTML input

### PR DESCRIPTION
**Pull request by aleks2407**
Closes gh-9

### CODE REVIEW

File: scripts/convert-to-celsius.js [hunk 1]
```diff
@@ -3,8 +3,31 @@
 console.log('-- loading: convertToCelsius');
 
 
-function convertToCelsius() { }
+function convertToCelsius(f) { 
+  return (f-32) * 5/9;
+}
+
```
Comment (mametur) [inline scripts/convert-to-celsius.js:9]: `const _2_arg = 14;
  const _2_expect = -10;
  const _2_actual = convertToC(_2_arg);
  console.assert(_2_actual === _2_expect, 'Test 2');`

Can you change this  `const _2_actual = convertToC(_2_arg);`  as  `const _2_actual = convertToCelsius(_2_arg);`

**Review by mametur — CHANGES_REQUESTED** [review]
`convertToC()`  function throws an error bcs it is not defined

File: scripts/convert-to-celsius.js (hunk 1 above)
Comment (aleks2407) [reply scripts/convert-to-celsius.js:9]: My function doesn't work with this test. I did it differently. Please check the code. To make it different from the "confirm-ending" function, I wrote a function to convert the temperature.

Comment (aleks2407) [reply scripts/convert-to-celsius.js:9]: My function doesn't work with this test. I did it differently. Please check the code. To make it different from the "confirm-ending" function, I wrote a function to convert the temperature.

Comment (aleks2407) [reply scripts/convert-to-celsius.js:9]: Sorry, i saw it now. Will correct immediately

**Review by mametur — APPROVED** [review]

**Review by mametur — APPROVED** [review]

**Review by mametur — APPROVED** [review]

**Pull request merged**